### PR TITLE
Do not request too much data when performing searches.

### PR DESCRIPTION
**Pull request by hatched**
We were requesting way to much data in searches. This reduces search speed increase from ~4s to 500ms for "apache"

### CODE REVIEW

**Comment by hatched** [comment]
#### To QA

Perform searches, see that they are faster.
All of the same information is visible in the search results and in the user profile.
No errors in browser console.

File: jujugui/static/gui/src/app/components/search-results/search-results.js [hunk 1]
```diff
@@ -142,7 +142,7 @@ YUI.add('search-results', function(Y) {
       // Parse the raw results.
       if (error) {
         this._changeActiveComponent('error');
-        console.error('Search request failed.');
+        console.error('Search request failed.', error);
```
Comment (frankban) [inline jujugui/static/gui/src/app/components/search-results/search-results.js]: `console.error('search request failed:', error);` ?

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7337/

File: jujugui/static/gui/src/app/components/search-results/search-results.js (hunk 1 above)
Comment (hatched) [inline jujugui/static/gui/src/app/components/search-results/search-results.js]: changed

**Comment by hatched** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**